### PR TITLE
chore: use localization files in console for sql review

### PR DESCRIPTION
**Pull request by ecmadao**
Pending https://github.com/bytebase/bytebase/pull/1801

### CODE REVIEW

File: scripts/get_sql_review_configuration.sh [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+# TODO: use version to fetch the lastest file
+FILE="sqlReviewConfig.yaml"
+URL="https://raw.githubusercontent.com/bytebase/bytebase/main/frontend/src"
+
+curl -o "./common/$FILE" "$URL/types/$FILE"
+
+echo "schema review configuration file updated."
```
Comment (tianzhou) [inline scripts/get_sql_review_configuration.sh:8]:
```suggestion
echo "Successfully updated SQL Review configuration file."
```

File: scripts/get_sql_review_configuration.sh [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+LOCALIZATION_FOLDER="./locales/sql-review"
+mkdir -p $LOCALIZATION_FOLDER
+
+curl -o "$LOCALIZATION_FOLDER/en.json" "$URL/locales/sql-review/en-US.json"
+curl -o "$LOCALIZATION_FOLDER/zh.json" "$URL/locales/sql-review/zh-CN.json"
+
+echo "schema review localization file updated."
```
Comment (tianzhou) [inline scripts/get_sql_review_configuration.sh:16]:
```suggestion
echo "Successfully updated SQL Review localization file."
```

File: package.json [hunk 3]
```diff
@@ -7,7 +7,7 @@
     "build": "npm run fetch-file && nuxt build",
     "start": "nuxt start",
     "generate": "npm run fetch-file && nuxt generate",
-    "fetch-file": "sh ./scripts/get_schema_review_rules.sh"
+    "fetch-file": "sh ./scripts/get_sql_review_configuration.sh"
```
Comment (tianzhou) [inline package.json:10]: fetch-sql-review-file

and name the script "fetch_sql_review_file.sh" ?

**Review by tianzhou — DISMISSED** [review]
LGTM

**Comment by tianzhou** [comment]
BTW, there is an error might be related to the recent PR

CleanShot 2022-07-09 at 01 24 12@2x

**Comment by ecmadao** [comment]
> BTW, there is an error might be related to the recent PR
> 
> CleanShot 2022-07-09 at 01 24 12@2x

@tianzhou  Is this error happening on your local machine? We will fetch SQL review files from the console every time running the `npm run dev/build/generate`, so it looks like the curl failed, maybe because of the proxy or network issue.

**Comment by tianzhou** [comment]
> > BTW, there is an error might be related to the recent PR
> > CleanShot 2022-07-09 at 01 24 12@2x
> 
> @tianzhou Is this error happening on your local machine? We will fetch SQL review files from the console every time running the `npm run dev/build/generate`, so it looks like the curl failed, maybe because of the proxy or network issue.

I see

**Review by tianzhou — APPROVED** [review]
LGTM

**Pull request merged**